fleet-management-api/src/main/java/cz/fi/muni/pa165/dto/InspectionDTO.java

```diff
@@ -24,6 +24,8 @@ public Long getId() {
         return this.id;
     }
 
+    public void setId(Long id) {this.id = id;}
```

Comment:
I think that `setId()` is a code smell - there is really no reason to change id of any object during its lifecycle, so it should be in constructor. But sidne DTO does not contain any logic and they use in refference project, I will let it pass.